amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -11,15 +11,15 @@ import AppConfig from 'config/config';
 import ColumnDescEditableText from 'components/TableDetail/ColumnDescEditableText';
 import { GlobalState } from 'ducks/rootReducer';
 import { logClick } from 'ducks/utilMethods';
-import { ToggleRequestAction } from 'ducks/notification/types';
+import { OpenRequestAction } from 'ducks/notification/types';
 import { openRequestDescriptionDialog } from 'ducks/notification/reducer';
-import { TableColumn } from 'interfaces';
+import { RequestMetadataType, TableColumn } from 'interfaces';
 
 // TODO: Use css-modules instead of 'import'
 import './styles.scss';
 
 interface DispatchFromProps {
-  openRequestDescriptionDialog: (checkedInputs: string[]) => ToggleRequestAction;
+  openRequestDescriptionDialog: (requestMetadataType: RequestMetadataType, columnIndex: number) => OpenRequestAction;
```

Comment:
Still not a big fan of 'columnIndex', especially considering that sorting and filtering columns are coming up. We could probably pass an entire `TableColumn` or even just the `name`.

Reply:
I can go with the `name`. The thinking here was to avoid passing a repetitive chunks of data.